core/src/main/scala/chisel3/internal/firrtl/Serializer.scala

```diff
@@ -29,10 +29,12 @@ private[chisel3] object Serializer {
     }
 
   /** Generate a legal FIRRTL name. */
-  private def legalize(name: String): String = name match {
-    // If the name starts with a digit, then escape it with backticks.
-    case _ if name.head.isDigit => s"`$name`"
-    case _                      => name
+  private def legalize(name: String): String = {
+    name match {
+      // If the name starts with a digit, then escape it with backticks.
+      case _ if name.head.isDigit => s"`$name`"
+      case _                      => name
+    }
   }
```

Comment:
This change feels pointless but also is fine

Reply:
Not intentional. I think I was using this for debugging at one point and didn't change it back (do a self-review). Fixed in: d207464b7